packages/common/src/components/Filter/EnumFilter.tsx

```diff
@@ -116,56 +119,67 @@ export const EnumFilter = ({
   filterId,
   showFilter = true,
   resolvedLanguage,
-  hasInlineFilter = false,
-}: FilterTypeProps & InlineFilter) => {
-  const [isExpanded, setExpanded] = useState(false);
+}: FilterTypeProps) => {
+  const [isOpen, setIsOpen] = useState(false);
   const { uniqueEnumLabels, onUniqueFilterUpdate, selectedUniqueEnumLabels } = useUnique({
     supportedEnumValues,
     onSelectedEnumIdsChange,
     selectedEnumIds,
     resolvedLanguage,
   });
 
-  const deleteFilter = (label: string | ToolbarChip | SelectOptionObject): void =>
+  const deleteFilter = (label: string | ToolbarChip): void =>
     onUniqueFilterUpdate(selectedUniqueEnumLabels.filter((filterLabel) => filterLabel !== label));
 
-  const hasFilter = (label: string | SelectOptionObject): boolean =>
+  const hasFilter = (label: string): boolean =>
     !!selectedUniqueEnumLabels.find((filterLabel) => filterLabel === label);
 
-  const addFilter = (label: string | SelectOptionObject): void => {
+  const addFilter = (label: string): void => {
     if (typeof label === 'string') {
       onUniqueFilterUpdate([...selectedUniqueEnumLabels, label]);
     }
   };
 
-  const options = uniqueEnumLabels.map((label) => <SelectOption key={label} value={label} />);
-
-  const onFilter: (
-    event: React.ChangeEvent<HTMLInputElement> | null,
-    textInput: string,
-  ) => React.ReactElement[] | undefined = (_event, textInput) => {
-    if (textInput === '') {
-      return options;
-    }
-    const filtered = options.filter((item) => {
-      return item.key.toString().toLowerCase().includes(textInput.toLowerCase());
-    });
-    return filtered;
+  const onToggleClick = () => {
+    setIsOpen(!isOpen);
   };
 
-  const onSelect: (
-    event: SelectEventType,
-    value: SelectValueType,
-    isPlaceholder?: boolean,
-  ) => void = (_event, value, isPlaceholder) => {
-    if (isPlaceholder) {
-      return;
-    }
-    hasFilter(value) ? deleteFilter(value) : addFilter(value);
+  const onSelect = (
+    _event: React.MouseEvent<Element, MouseEvent> | undefined,
+    value: string | number | undefined,
+  ) => {
+    hasFilter(value as string) ? deleteFilter(value as string) : addFilter(value as string);
   };
 
-  const onToggle: (isExpanded: boolean, event: ToggleEventType) => void = (isExpanded) => {
-    setExpanded(isExpanded);
+  const toggle = (toggleRef: React.Ref<MenuToggleElement>) => (
+    <MenuToggle
+      ref={toggleRef}
+      onClick={onToggleClick}
+      isExpanded={isOpen}
+      style={
```

Comment:
why inline styling?

Reply:
It doesn't work with className since the `pf-v5-c-menu-toggle` style overwrites it. 
I replacing it with the `isFullWidth` prop.